build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
         classpath 'com.android.tools.build:gradle:4.2.2'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "androidx.navigation:navigation-safe-args-gradle-plugin:2.4.0-alpha04"
+        classpath "com.google.dagger:hilt-android-gradle-plugin:2.37"
```

Comment:
We can use the greatest version which is `2.38` I guess.

Reply:
Done. 6ace771